recipes/keepalive/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "keepalive" %}
+{% set version = "0.5" %}
+{% set sha256 = "3c6b96f9062a5a76022f0c9d41e9ef5552d80b1cadd4fccc1bf8f183ba1d1ec1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - keepalive
+
+about:
+  home: https://github.com/wikier/keepalive
+  license: LGPL-2.1
+  license_file: LICENSE
```

Comment:
@satra -- since the LICENSE isn't included in the source tarball feel free to drop this line.